### ISSUE TITLE
ResNet50 pretrained model doesn't work well

### ISSUE THREAD

**Issue by Summer-Summer** [caffe2/models#32]
I am really confused about this pretrianed model 
I use the ILSVRC2012 validation set to test the pretrained models (AlexNet GoogleNet VGG)
They are  just pretty good
When it comes to ResNet50,accuary is 0

Or something wrong i have done?
I am really lookforward to answers.

**Comment by houseroad** [comment]
The reason and solution is here:
https://github.com/onnx/models/issues/64
